### PR TITLE
Custom theme

### LINKED ISSUE #3: Add Custom Theme and update boilerplate
Adding of custom color, font and global style that will be used in the project

### PR DESCRIPTION
**Pull request by ruddi10**
fixes #3

### CODE REVIEW

**Comment by ruddi10** [comment]
@muskankhedia  @aquibbaig PTAL

**Comment by aquibbaig** [comment]
@ruddi10 For initials, please run a `yarn format` and then fix the following warnings:
```
src/layouts/BasicLayout.tsx
  Line 2:27:  'useState' is defined but never used  @typescript-eslint/no-unused-vars

src/theme/global.tsx
  Line 4:13:  Missing return type on function                       @typescript-eslint/explicit-module-boundary-types
  Line 4:14:  Argument 'props' should be typed with a non-any type  @typescript-eslint/explicit-module-boundary-types
  Line 4:21:  Unexpected any. Specify a different type              @typescript-eslint/no-explicit-any
```

**Comment by ruddi10** [comment]
> @ruddi10 For initials, please run a `yarn format` and then fix the following warnings:
> 
> ```
> src/layouts/BasicLayout.tsx
>   Line 2:27:  'useState' is defined but never used  @typescript-eslint/no-unused-vars
> 
> src/theme/global.tsx
>   Line 4:13:  Missing return type on function                       @typescript-eslint/explicit-module-boundary-types
>   Line 4:14:  Argument 'props' should be typed with a non-any type  @typescript-eslint/explicit-module-boundary-types
>   Line 4:21:  Unexpected any. Specify a different type              @typescript-eslint/no-explicit-any
> ```

Have fixed this

**Review by aquibbaig — CHANGES_REQUESTED** [review]
The files inside `theme` directory can be simple typescript files since they do not include any JSX

File: src/theme/branding.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const branding = {
+  colors: {
+    lightPrimary: "#e5e5e5",
+    lightSecondary: "#f7f7f7",
+    lightText: "#666666",
+    darkPrimary: "#292929",
+    darkSecondary: "#121212",
```
Comment (aquibbaig) [inline src/theme/branding.tsx:7]: Separate background variables from text variables using consistent naming

**Review by aquibbaig — CHANGES_REQUESTED** [review]

**Review by muskankhedia — APPROVED** [review]
LGTM!

**Pull request merged**